staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -691,7 +694,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				Returns(http.StatusOK, "OK", producedObject).
 				Reads(metav1.Patch{}).
 				Writes(producedObject)
-			if err := addObjectParams(ws, route, versionedUpdateOptions); err != nil {
+			if err := addObjectParams(ws, route, versionedPatchOptions); err != nil {
```

Comment:
The `dryRun` query parameter is removed from generated openapi spec.

/hold

I realized that `addObjectParams` only parses top-level non-struct fields. `dryRun` gets dropped because UpdateOptions is nested in PatchOptions. You may want to keep `addObjectParams(ws, route, versionedUpdateOptions)` to reflect both `dryRun` and `force` query parameters. 
https://github.com/kubernetes/kubernetes/blob/e3ddaaad1d66ce1f74d6fa809d834c4de64546f9/staging/src/k8s.io/apiserver/pkg/endpoints/installer.go#L912-L914

Reply:
Fixed, feel free to PTAL but as far as I can tell, the swagger looks better :-)